src/main/java/com/google/codeu/servlets/ChartServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.codeu.servlets;
+
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import java.util.Scanner;
+
+
+@WebServlet("/bookchart")
+public class ChartServlet extends HttpServlet {
+
+
+  private JsonArray bookRatingArray;
+
+  // create file to help with serializing the data
+  private static class bookRating {
```

Comment:
I believe that class names are always capitalized. You should do that here for this inner class.

Reply:
I'm following the starter code convention, though. Since this is a "helper" class, I'm not sure it needs to be.

Reply:
Alright mb